bemani/api/objects/catalog.py

```diff
@@ -155,11 +155,30 @@ def __format_jubeat_extras(self) -> Dict[str, List[Dict[str, Any]]]:
         else:
             return {"emblems": []}
 
+    def __format_iidx_extras(self) -> Dict[str, List[Dict[str, Any]]]:
```

Comment:
This should go along with an API documentation change in this section: https://github.com/DragonMinded/bemapi#iidx-additional-attributes-and-documentation-2

Much like SDVX has documentation for its own extra catalog entries.

Reply:
Pull request submitted